controlplane/kubeadm/internal/etcd_client_generator.go

```diff
@@ -38,18 +36,17 @@ type etcdClientGenerator struct {
 func (c *etcdClientGenerator) forNode(ctx context.Context, name string) (*etcd.Client, error) {
```

Comment:
Can we combine forNode() with forAny()? If caller wants to connect to a particular node, it can send a single node in the NodeList  argument?

Reply:
updated. Thanks for the suggestion